11_webui.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/bash
+
+set -eux
+source common.sh
+
+figlet "Deploying web ui" | lolcat
+eval "$(go env)"
+
+
+export UIPATH="$GOPATH/src/github.com/kubevirt/web-ui-operator"
+
+oc new-project kubevirt-web-ui
+cd $UIPATH/web-ui-operator/deploy
+oc apply -f service_account.yaml
+oc adm policy add-scc-to-user anyuid -z kubevirt-web-ui-operator
+oc adm policy add-cluster-role-to-user cluster-admin system:serviceaccount:kubevirt-web-ui:kubevirt-web-ui-operator
+oc apply -f role.yaml
+oc apply -f role_binding.yaml
+oc apply -f role_binding_kube-public.yaml
+oc apply -f crds/kubevirt_v1alpha1_kwebui_crd.yaml
+oc apply -f operator.yaml
```

Comment:
Can all these manifests be combined into a single file under `manifests/`?

Reply:
i dont see too much value of the value it brings...

Reply:
I think the main value will be it's going to be easier to move to kni-installer - pretty soon we'll want to move all manifests either there, or to some other operator e.g MCO, we can't rely on these scripts forever, so IMO it's good to keep in mind the best long-term home for these pieces.

Not blocking on it for now, but it's good to start discussion on that topic I think.